LICENSE

```diff
@@ -1,4 +1,4 @@
-Copyright (c) 2016 JFrog Ltd.
+Copyright (c) 2019 JFrog Ltd.
```

Comment:
Should we update to the current year, or a range with initial year, like `2016-2019` ?